Doc/library/subprocess.rst

```diff
@@ -344,7 +344,8 @@ functions.
                  startupinfo=None, creationflags=0, restore_signals=True, \
                  start_new_session=False, pass_fds=(), *, group=None, \
                  extra_groups=None, user=None, umask=-1, \
-                 encoding=None, errors=None, text=None, pipesize=-1)
+                 encoding=None, errors=None, text=None, pipesize=-1, \
+                 setpgid=-1)
```

Comment:
Maybe use `None` as a default value (as for `group`, `user`, etc)? It would look more Pythonic.

Also note that `group` is used for `setregid()`, `extra_groups` for `setgroups()`, `user` for `setreuid()`, `start_new_session` for `setsid()`, etc. Should not more descriptive name be used for `setpgid()`. Maybe `process_group`?